test/test_sync.cpp

```diff
@@ -2708,8 +2712,8 @@ TEST(Sync_SSL_Certificate_1)
     session_config.verify_servers_ssl_certificate = true;
     session_config.ssl_trust_certificate_path = ca_dir + "crt.pem";
 
-    Session session = fixture.make_session(db, std::move(session_config));
-    fixture.bind_session(session, "/test", g_signed_test_user_token, ProtocolEnvelope::realms);
```

Comment:
Looks like you need to assign `g_signed_test_user_token` to `signed_user_token` in the session config above?
(there's also another instance on 2810)